test/thermo/thermoFromYaml.cpp

```diff
@@ -431,4 +446,4 @@ TEST(ThermoFromYaml, BinarySolutionTabulatedThermo)
     EXPECT_NEAR(thermo->entropy_mass(), 90.443481807823474, 1e-12);
     thermo->setMoleFractionsByName("Li[anode]: 0.55, V[anode]: 0.45");
     EXPECT_NEAR(thermo->gibbs_mass(), -87066.246182649265, 1e-9);
-}
+}
```

Comment:
Keep the newline at the end of the file.

Reply:
Corrected.

Reply:
Corrected.